usr/local/www/diag_dns.php

```diff
@@ -29,20 +29,24 @@
 */
 
 /*
-	pfSense_MODULE:	dns
+	pfSense_MODULE: dns
 */
 
-$pgtitle = array(gettext("Diagnostics"),gettext("DNS Lookup"));
+/* Bootstrap conversion: sbeaver@netgate.com */
+
+$pgtitle = array(gettext("Diagnostics"),gettext("DNS lookup"));
 require("guiconfig.inc");
 
 $host = trim($_REQUEST['host'], " \t\n\r\0\x0B[];\"'");
 $host_esc = escapeshellarg($host);
 
-if (is_array($config['aliases']['alias'])) {
-	$a_aliases = &$config['aliases']['alias'];
-} else {
-	$a_aliases = array();
+/* If this section of config.xml has not been populated yet we need to set it up 
+*/
+if (!is_array($config['aliases']['alias'])) {
```

Comment:
You mean don’t add comments?

That logic change and comment were added upstream to the pfSense master.

Steve

> On Mar 26, 2015, at 4:50 PM, SjonHortensius <notifications@github.com> wrote:
> 
> In usr/local/www/diag_dns.php <https://github.com/SjonHortensius/pfsense/pull/42#discussion_r27254529>:
> 
> >  require("guiconfig.inc");
> >  
> >  $host = trim($_REQUEST['host'], " \t\n\r\0\x0B[];\"'");
> >  $host_esc = escapeshellarg($host);
> >  
> > -if (is_array($config['aliases']['alias'])) {
> > -	$a_aliases = &$config['aliases']['alias'];
> > -} else {
> > -	$a_aliases = array();
> > +/* If this section of config.xml has not been populated yet we need to set it up 
> > +*/
> > +if (!is_array($config['aliases']['alias'])) {
> Please change only the layout, and logic if necessary. This specifically isn't necessary and only diffuses the diff
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SjonHortensius/pfsense/pull/42/files#r27254529>.
>